README.md

```diff
@@ -1 +1,35 @@
-# nRF-802.15.4-sniffer
\ No newline at end of file
+# nRF 802.15.4 sniffer
+
+This repository contains firmware and Wireshark extcap script that can be used with nRF52840 chip as an 802.15.4 sniffer.
+
+__Note:__ this project is experimental. There is __no Windows support__ in the current version.
+
+The software provided has been tested with the nRF52840-DK board and Wireshark 2.4.5 from the official Ubuntu 18.04 package.
+
+## Dependencies
+* Wireshark (Ubuntu package `wireshark`)
+* pySerial (Ubuntu package `python-serial` or `python3-serial`)
+
+## Quick start guide
+
+To start using the sniffer, you must flash the firmware, install the script, and configure the sniffer in Wireshark.
+
+### Flash firmware
+1. Connect the nRF52840-DK to the PC with an USB cable by connecting it to the J2 USB port.
+2. Flash the firmware with the following command:
+```
+nrfjprog -f nrf52 --program nrf802154_sniffer.hex --chiperase -r
+```
+
+### Install extcap script
+Copy the provided script to the appropriate directory:
+```
+sudo cp nrf802154_sniffer.py /usr/lib/x86_64-linux-gnu/wireshark/extcap/
```

Comment:
I think we should add a note that this directory can be found in wireshark `Help->About Wireshark->Folders` under position `Extcap path` as this may differ.

Reply:
That's right. We can improve this documentation in many ways. For now I would keep it simple and I encourage you to create PR with more details.